### PR TITLE
Add Bypassing UDP multicast to Motionblinds

### PR DESCRIPTION
**Pull request by starkillerOG**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/50067
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/motion_blinds.markdown [hunk 1]
```diff
@@ -111,3 +111,17 @@ If using separate VLANs, make sure the 238.0.0.18:32100 and 238.0.0.18:32101 por
 For some routers "IGMP snooping" on the used wireless interface needs to be disabled to let the IGMP/multicast messages through.
 
 For Ubiquiti routers/access points the "Enable multicast enhancement (IGMPv3)" should be disabled.
+
+### Bypassing UDP multicast
+
+If UDP Multicast does not work in your setup (due to network limitations), this integration can be used in locall polling mode.
```
Comment (MartinHjelmare) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
If UDP Multicast does not work in your setup (due to network limitations), this integration can be used in local polling mode.
```

File: source/_integrations/motion_blinds.markdown [hunk 2]
```diff
@@ -111,3 +111,17 @@ If using separate VLANs, make sure the 238.0.0.18:32100 and 238.0.0.18:32101 por
...
+Go to Settings -> Integrations -> on the already set up Motion Blinds integration click "configure" --> disable the "Wait for push" option (disabled by default).
+
+The default update interval of the Motion Blinds integration is each 10 minutes, when UDP multicast pushes do not work, this polling intervall can be a bit high.
```
Comment (MartinHjelmare) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
The default update interval of the Motion Blinds integration is every 10 minutes. When UDP multicast pushes do not work, this polling interval can be a bit high.
```

File: source/_integrations/motion_blinds.markdown [hunk 3]
```diff
@@ -111,3 +111,17 @@ If using separate VLANs, make sure the 238.0.0.18:32100 and 238.0.0.18:32101 por
...
+To increase the polling intervall:
```
Comment (MartinHjelmare) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
To increase the polling interval:
```

File: source/_integrations/motion_blinds.markdown [hunk 4]
```diff
@@ -111,3 +111,17 @@ If using separate VLANs, make sure the 238.0.0.18:32100 and 238.0.0.18:32101 por
...
+Go to Settings -> Integrations -> on the already set up Motion Blinds integration click more options (three dots) and select "systemoptions" -> disable "polling for updates".
```
Comment (MartinHjelmare) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
Go to Settings -> Integrations -> on the already set up Motion Blinds integration click more options (three dots) and select "System options" -> disable "polling for updates".
```

File: source/_integrations/motion_blinds.markdown [hunk 5]
```diff
@@ -111,3 +111,17 @@ If using separate VLANs, make sure the 238.0.0.18:32100 and 238.0.0.18:32101 por
...
+Now create an automation with as trigger a timepattern and select your desired polling time.
```
Comment (MartinHjelmare) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
Now create an automation with as trigger a time pattern and select your desired polling time.
```

File: source/_integrations/motion_blinds.markdown [hunk 6]
```diff
@@ -111,3 +111,17 @@ If using separate VLANs, make sure the 238.0.0.18:32100 and 238.0.0.18:32101 por
...
+As the action select "call service" and select "update entity", select one of the motion blinds covers as entity.
```
Comment (MartinHjelmare) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
As the action select "Call service" and select "Update entity", select one of the motion blinds covers as entity.
```

**Comment by starkillerOG** [comment]
sorry for all the spelling mistakes

File: source/_integrations/motion_blinds.markdown [hunk 7]
```diff
@@ -111,3 +111,28 @@ If using separate VLANs, make sure the 238.0.0.18:32100 and 238.0.0.18:32101 por
...
+Go to Settings -> Integrations -> on the already set up Motion Blinds integration click "configure" --> disable the "Wait for push" option (disabled by default).
+
+The default update interval of the Motion Blinds integration is every 10 minutes. When UDP multicast pushes do not work, this polling interval can be a bit high.
+To increase the polling interval:
+Go to Settings -> Integrations -> on the already set up Motion Blinds integration click more options (three dots) and select "System options" -> disable "polling for updates".
+Now create an automation with as trigger a time pattern and select your desired polling time.
+As the action select "Call service" and select "Update entity", select one of the motion blinds covers as entity.
+You only have to create one automation with only one motion blind cover as entity, the rest will update at the same time.
+
+Example YAML automation for custom polling intervall (every minute):
```
Comment (MartinHjelmare) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
Example YAML automation for custom polling interval (every minute):
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks! :+1:

**Pull request merged**